README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Build Status](https://travis-ci.org/roots/trellis.svg)](https://travis-ci.org/roots/trellis)
 
-Trellis is a set of [Ansible](http://www.ansible.com/home) [playbooks](http://docs.ansible.com/playbooks.html) to automatically configure servers and deploy WordPress sites. It easily creates development environments with Vagrant to help achieve development & production parity.
+Trellis is a set of [Ansible](http://www.ansible.com/home) [playbooks](http://docs.ansible.com/playbooks.html) to automatically configure servers and deploy WordPress sites. It easily creates development environments with Vagrant for better development & production parity.
```

Comment:
I don't agree with "for better". Either keep as-is or change entirely:

Trellis is a set of Ansible playbooks for creating and deploying WordPress server environments. Use Trellis to create local development environments with Vagrant as well as robust production and staging environments on remote servers.

Or something close to that.